### PR TITLE
Update TiSpark documentation

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)
- Delete get-started-with-tispark.md
- Update  tispark-overview.md

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/6459
https://github.com/pingcap/docs/pull/8205
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tispark-overview.md [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
+
+- TiSpark：数据分析和 ETL
+- TiKV：数据检索
+- 调度系统：生成报表
+
+除此之外，TiSpark 还提供了分布式写入 TiKV 的功能。与使用 Spark 结合 JDBC 写入 TiDB 的方式相比，分布式写入 TiKV 能够实现事务（要么全部数据写入成功，要么全部都写入失败），而且写入速度更快。
+
+> **警告：**
+>
+> 由于 TiSpark 直接访问 TiKV，所以 TiDB Server 使用的访问控制机制并不适用于 TiSpark。
```
Comment (xuanyu66) [inline tispark-overview.md:33]: 可以加上目前 TiSpark 实现了部分鉴权与授权功能，具体请看`安全` 那一章

**Review by xuanyu66 — CHANGES_REQUESTED** [review]

File: tispark-overview.md [hunk 2]
```diff
@@ -1,25 +1,36 @@
...
+[TiSpark](https://github.com/pingcap/tispark) 是 PingCAP 为解决用户复杂 OLAP 需求而推出的产品。它借助 Spark 平台，同时融合 TiKV 分布式集群的优势，和 TiDB 一起为用户一站式解决 HTAP (Hybrid Transactional/Analytical Processing) 的需求。
 
-## 概述
+[TiFlash](/tiflash/tiflash-overview.md) 也是一个解决 HTAP 需求的产品。TiFlash 和 TiSpark 都允许使用多个主机在 OLTP 数据上执行 OLAP 查询。TiFlash 是列式存储，这提供了更高效的分析查询。TiFlash 和 TiSpark 可以同时使用。
 
-TiSpark 是将 Spark SQL 直接运行在分布式存储引擎 TiKV 上的 OLAP 解决方案。其架构图如下：
+TiSpark 依赖于 TiKV 集群和 Placement Driver (PD)，也需要你搭建一个 Spark 集群。本文简单介绍如何部署和使用 TiSpark。本文假设你对 Spark 有基本认知。你可以参阅 [Apache Spark 官网](https://spark.apache.org/docs/latest/index.html)了解 Spark 的相关信息。
 
-![TiSpark Architecture](/media/tispark-architecture.png)
+TiSpark 深度整合了 Spark Catalyst 引擎，可以对计算进行精确地控制，使 Spark 能够高效地读取 TiKV 中的数据。TiSpark 还提供了索引支持以实现高速的点查。
```
Comment (shichun-0415) [inline tispark-overview.md:17]:
```suggestion
TiSpark 深度整合了 Spark Catalyst 引擎，可以对计算进行精确地控制，使 Spark 能够高效地读取 TiKV 中的数据。TiSpark 还提供索引查找，帮助实现高速点查。
```
```suggestion
TiSpark 深度整合了 Spark Catalyst 引擎，可以对计算进行精确地控制，使 Spark 能够高效地读取 TiKV 中的数据。TiSpark 还提供了索引支持以实现高速的点查。
```

Comment (shichun-0415) [inline tispark-overview.md:17]:
```suggestion
TiSpark 深度整合了 Spark Catalyst 引擎，可以对计算进行精确地控制，使 Spark 能够高效地读取 TiKV 中的数据。TiSpark 还提供索引查找，帮助实现高速点查。
```
```suggestion
TiSpark 深度整合了 Spark Catalyst 引擎，可以对计算进行精确地控制，使 Spark 能够高效地读取 TiKV 中的数据。TiSpark 还提供了索引支持以实现高速的点查。
```

File: tispark-overview.md [hunk 3]
```diff
@@ -1,25 +1,36 @@
...
 
-+ TiSpark 深度整合了 Spark Catalyst 引擎，可以对计算提供精确的控制，使 Spark 能够高效的读取 TiKV 中的数据，提供索引支持以实现高速的点查。
-+ 通过多种计算下推减少 Spark SQL 需要处理的数据大小，以加速查询；利用 TiDB 的内建的统计信息选择更优的查询计划。
-+ 从数据集群的角度看，TiSpark + TiDB 可以让用户无需进行脆弱和难以维护的 ETL，直接在同一个平台进行事务和分析两种工作，简化了系统架构和运维。
-+ 用户借助 TiSpark 项目可以在 TiDB 上使用 Spark 生态圈提供的多种工具进行数据处理。例如，使用 TiSpark 进行数据分析和 ETL；使用 TiKV 作为机器学习的数据源；借助调度系统产生定时报表等等。
-+ 除此之外，TiSpark 还提供了分布式写入 TiKV 的功能。相比使用 Spark 结合 JDBC 的方式写入 TiDB，分布式写入 TiKV 可以实现事务（要么全部数据写入成功，要么全部都写入失败），并且写入速度会更快。
+TiSpark 通过将计算下推到 TiKV 中提升了数据查询的效率，减少了 Spark SQL 需要处理的数据大小，通过利用 TiDB 内置的统计信息选择更优的查询计划。
```
Comment (shichun-0415) [inline tispark-overview.md:19]:
```suggestion
TiSpark 通过将计算下推到 TiKV 中提升了数据查询的效率，减少了 Spark SQL 需要处理的数据量，通过利用 TiDB 内置的统计信息选择最优查询计划。
```

File: tispark-overview.md [hunk 4]
```diff
@@ -33,225 +44,233 @@ TiSpark 是将 Spark SQL 直接运行在分布式存储引擎 TiKV 上的 OLAP 
...
+#### 下载并安装
 
-```
-spark.tispark.pd.addresses=$your_pd_servers
-spark.sql.extensions=org.apache.spark.sql.TiExtensions
-```
+你可以从 [Apache Spark Archive](https://archive.apache.org/dist/spark/) 下载 Spark 2.x 版本。
 
-`your_pd_servers` 是用逗号分隔的 PD 地址，每个地址使用 `地址:端口` 的格式。
+如果你不需要 Hadoop 支持，请选择 Spark **2.4.x** 版本、带有任意版本 Hadoop 依赖的预编译二进制包，例如 `spark-2.4.8-bin-hadoop2.7.tgz`。如果需要使用 Hadoop 集群（如 Hadoop 2.6 版本），则选择相应的 Hadoop 版本号，例如 `spark-2.4.8-bin-hadoop2.6.tgz`。对于 Hadoop 2.x 之前的版本，你可以从源代码[自行构建](https://spark.apache.org/docs/latest/building-spark.html)。
```
Comment (shichun-0415) [inline tispark-overview.md:80]: 原文是 hadoop 2.7

File: tispark-overview.md [hunk 5]
```diff
@@ -33,225 +44,233 @@ TiSpark 是将 Spark SQL 直接运行在分布式存储引擎 TiKV 上的 OLAP 
...
+在 `spark-defaults.conf` 文件中添加如下内容：
 
-你可以在 [Download Apache Spark™ 页面](https://spark.apache.org/downloads.html)下载 Apache Spark。
+```
+spark.tispark.pd.addresses $pd_host:$pd_port
+spark.sql.extensions org.apache.spark.sql.TiExtensions
+```
 
-对于 Standalone 模式且无需 Hadoop 支持，则选择 Spark 2.3.x 或者 Spark 2.4.x 且带有 Hadoop 依赖的 Pre-build with Apache Hadoop 2.x 任意版本。如有需要配合使用的 Hadoop 集群，则选择对应的 Hadoop 版本号。你也可以选择从源代码[自行构建](https://spark.apache.org/docs/latest/building-spark.html)以配合官方 Hadoop 2.x 之前的版本。
+其中 `spark.tispark.pd.addresses` 允许输入按逗号 (',') 分隔的多个 PD 服务器，请指定每个服务器的端口号。例如，当你的多个 PD 服务器在 `10.16.20.1,10.16.20.2,10.16.20.3` 的 2379 端口上时，将配置 `spark.tispark.pd.addresses` 为 `10.16.20.1:2379,10.16.20.2:2379,10.16.20.3:2379`。
```
Comment (shichun-0415) [inline tispark-overview.md:126]:
```suggestion
其中 `spark.tispark.pd.addresses` 允许输入按逗号 (',') 分隔的多个 PD 服务器，请指定每个服务器的端口号。例如，当你有多个 PD 服务器在 `10.16.20.1,10.16.20.2,10.16.20.3` 的 2379 端口上时，将配置 `spark.tispark.pd.addresses` 为 `10.16.20.1:2379,10.16.20.2:2379,10.16.20.3:2379`。
```

File: tispark-overview.md [hunk 6]
```diff
@@ -33,225 +44,233 @@ TiSpark 是将 Spark SQL 直接运行在分布式存储引擎 TiKV 上的 OLAP 
...
-beeline> !connect jdbc:hive2://localhost:10000
-```
+首先，通过如下命令启用 beeline：
 
-{{< copyable "" >}}
+{{< copyable "shell-regular" >}}
 
 ```shell
-1: jdbc:hive2://localhost:10000> use testdb;
+./bin/beeline jdbc:hive2://localhost:10000
```
Comment (shichun-0415) [inline tispark-overview.md:260]: 英文下面还有这一行：Beeline version 1.2.2 by Apache Hive

File: tispark-overview.md [hunk 7]
```diff
@@ -33,225 +44,233 @@ TiSpark 是将 Spark SQL 直接运行在分布式存储引擎 TiKV 上的 OLAP 
 
 TiSpark 可以在 YARN，Mesos，Standalone 等任意 Spark 模式下运行。
 
-## 推荐配置
+本部分描述了 TiKV 与 TiSpark 集群分开部署、Spark 与 TiSpark 集群独立部署，以及TiKV 与 TiSpark 集群混合部署的建议配置。
```
[the remark this answers is not in the document]
Comment (Oreoxmt) [reply tispark-overview.md:47]: @shichun-0415 https://github.com/pingcap/docs/blob/master/tispark-overview.md?plain=1 跟这里的版本是一致的

File: tispark-overview.md (hunk 4 above)
Comment (Oreoxmt) [reply tispark-overview.md:80]: > 原文是 hadoop 2.7

这里我添加了一个 Hadoop 2.6 的示例是想要让“如果需要使用 Hadoop 集群，则选择相应的 Hadoop 版本号” 这句话比较容易理解，也就是需要使用的 Hadoop 的版本号体现在 `spark-2.4.8-bin-hadoop2.6.tgz` 上。
因为前一句的示例是 `spark-2.4.8-bin-hadoop2.6.tgz`，为了区分所以我用了 2.6

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by xuanyu66 — APPROVED** [review]

**Comment by xuanyu66** [comment]
@shiyuhang0 PTAL

**Review by shiyuhang0 — COMMENTED** [review]
TiSpark 2.5 推荐使用 catalog 的方式，TiSpark 未来版本将强制使用 catalog。
建议这个文档中也加入 catalog 模式的使用方式

File: tispark-overview.md [hunk 8]
```diff
@@ -33,225 +44,239 @@ TiSpark 是将 Spark SQL 直接运行在分布式存储引擎 TiKV 上的 OLAP 
...
 
-### 已有 Spark 集群的部署方式
+> **注意：**
+>
+> TiSpark 2.1.x 及更早版本的 jar 文件名形如 `tispark-core-2.1.9-spark_2.4-jar-with-dependencies.jar`。请在 [TiSpark Releases](https://github.com/pingcap/tispark/releases) 中确认你需要的 TiSpark 版本的 jar 文件名。
 
-如果在已有 Spark 集群上运行 TiSpark，无需重启集群。可以使用 Spark 的 `--jars` 参数将 TiSpark 作为依赖引入：
+以下是 TiSpark 2.4.1 版本 jar 包的安装示例：
 
 {{< copyable "shell-regular" >}}
```
[the remark this answers is not in the document]
Comment (Oreoxmt) [reply tispark-overview.md:104]: @shiyuhang0 {{< copyable "shell-regular" >}} 是文档站前端渲染程序需要的 code snippet，在渲染时会自动在代码块前添加命令前缀（$、#等），以及用户可以通过点击 Copy 方式复制代码块。

**Review by shiyuhang0 — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**